src/components/GradesTab/EditModal/HistoryHeader.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * HistoryHeader
+ * simple display container for an individual history table header
+ * @param {string} id - header id
+ * @param {string} label - header label
+ * @param {string} value - header value
+ */
+const HistoryHeader = ({ id, label, value }) => (
+  <div>
```

Comment:
Proposed refactor: make this a `<table>` or `grid` instead of `<div>`s masquerading poorly as a table.

Reply:
I would put that as a new ticket.  We also have to updgrade both our tables to be using DataTable anyway, so makes sense to put some thought into how we wanna do that.